docs.html

```diff
@@ -0,0 +1,552 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>👷Builder UI👷</title>
+    <link rel="stylesheet" href="styles/styles.css">
+    <link rel="stylesheet" href="./styles/components/styles.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+
+</head>
+
+<body>
+    <nav class="nav">
+        <div class="navbar-brand">
+            <h2 class="brand-name"><a href="/index.html">👷BuilderUI👷</a></h2>
+        </div>
+        <div class="navbar-links">
+            <ul class="navbar-links-list">
+                <li class="navbar-links-list-item"><a href="/">Home</a></li>
+                <li class="navbar-links-list-item"><a href="/docs.html">Docs</a></li>
+                <li class="navbar-links-list-item"><a href="https://github.com/shanudjn/builder-ui">Github</a></li>
+            </ul>
+        </div>
+    </nav>
+    <aside class="aside">
+
+        <ul>
+            <li class="aside-list-item"><a href="#installation"><b>Installation</b></a></li>
+
+            <li class="aside-list-item"><a href="#alert">Alert</a></li>
+            <li class="aside-list-item"><a href="#avatar">Avatar</a></li>
+            <li class="aside-list-item"><a href="#badge">Badge</a></li>
+            <li class="aside-list-item"><a href="#cards">Cards</a></li>
+            <li class="aside-list-item"><a href="#grids">Grids</a></li>
+            <li class="aside-list-item"><a href="#input">Input</a></li>
+            <li class="aside-list-item"><a href="#list">Lists</a></li>
+            <li class="aside-list-item"><a href="#navigation">Navigation</a></li>
+            <li class="aside-list-item"><a href="#ratings">Ratings</a></li>
+            <li class="aside-list-item"><a href="#text">Text</a></li>
+        </ul>
+    </aside>
```

Comment:
The left-navigation bar can be fixed to the side using `sticky` or `fixed` positioning so it's easier for the viewer of the site to switch between different pages.